plugins/scaffolder/src/components/TemplatePage/TemplatePage.tsx

```diff
@@ -54,7 +54,7 @@ const useTemplate = (
 
 const OWNER_REPO_SCHEMA = {
   $schema: 'http://json-schema.org/draft-07/schema#' as const,
-  required: ['storePath', 'owner'],
+  required: ['storePath', 'owner', 'access'],
```

Comment:
wondering if this should be optional as other providers might not need this?

Reply:
yep, I think leave this as optional for now/